specs/~dynamic-names.yml

```diff
@@ -18,17 +18,16 @@ overview: |
       ]
   The goal is to render the different types of items in different ways. The
   items having a key named `content` should be rendered with the template
-  `text.mustache` and the items having a key named `url` should be rendered
-  with the template `image.mustache`:
-  
+  `text.mustache`:
       {{!image.mustache}}
       <img src="{{url}}"/>
-  
+  And the items having a key named `url` should be rendered with the template
+  `image.mustache`:
       {{!text.mustache}}
       {{content}}
```

Comment:
I changed this section a bit because I found the code spacing a bit weird, now it should be better.

Reply:
The examples and the text are in opposite order, though. You show `image.mustache` where the text says "text.mustache" and vice versa.

Reply:
Sorry for the confusion, I will fix this in the next commit